openshift/templates/pipeline/pipeline-lup-api-build-deploy-dev.yaml

```diff
@@ -0,0 +1,124 @@
+﻿apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  creationTimestamp: '2022-02-11T19:39:02Z'
+  generation: 15
+  labels:
+    app: landuseplanning
+    env: dev
+    name: lup-api
+  managedFields:
+    - apiVersion: tekton.dev/v1beta1
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:metadata':
+          'f:labels':
+            .: {}
+            'f:app': {}
+            'f:env': {}
+            'f:name': {}
+        'f:spec':
+          .: {}
+          'f:params': {}
+          'f:resources': {}
+          'f:workspaces': {}
+      manager: Mozilla
+      operation: Update
+      time: '2022-02-11T19:39:23Z'
+    - apiVersion: tekton.dev/v1
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:spec':
+          'f:finally': {}
+          'f:tasks': {}
+      manager: Mozilla
+      operation: Update
+      time: '2025-03-21T20:10:27Z'
+  name: lup-api-build-deploy-dev
+  namespace: e8b9ad-tools
+  resourceVersion: '13745680169'
+  uid: 5b9acbbe-708b-4b3c-ac33-56abf3ebf0f2
+spec:
+  finally:
+    - name: push-to-teams-success
+      params:
+        - name: webhook-url
+          value: 'https://bcgov.webhook.office.com/webhookb2/85d42a52-f637-43bb-af02-817e93e8cb6f@6fdb5200-3d0d-4a8a-b036-d3685e359adc/IncomingWebhook/fb6c473cce1c4be49b6b54fc89d3e7c9/2c1af31d-c876-40e0-b2e6-ca14006d7319/V2qzaDPI4OCXq2xUDTmNdOacE3Qwdgo3SATHKcquR8K841'
+        - name: webhook-payload
+          value: |-
+            {
+                "@type": "MessageCard",
+                "@context": "https://schema.org/extensions",
+                "summary": "Issue 176715375",
+                "themeColor": "33cc33",
+                "title": "[LUP] Land Use Planning API has been deployed to dev",
+                "sections": [
+                    {
+                        "text": "The latest version of Land Use Planning **API** has been built and deployed to **dev**."
+                    }
+                ],
+                "potentialAction": [
+                    {
+                        "@type": "OpenUri",
+                        "name": "View on OpenShift",
+                        "targets": [
+                            {
+                                "os": "default",
+                                "uri": "https://console.apps.silver.devops.gov.bc.ca/k8s/ns/e8b9ad-tools/tekton.dev~v1beta1~Pipeline/lup-api-build-deploy-dev/Runs"
```

Comment:
Referencing the wrong pipeline yaml here. I'm not sure how this works with Tekton but we don't want to point to a version of the Pipeline that's using the old API if we can help it

Reply:
Fixed